File: Level1/year2016/year2016.java
package Level1.year2016;

class Solution {
    public String solution(int a, int b) {
        String[] arr = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        int[] month = {31,29,31,30,31,30,31,31,30,31,30,31};
        int day = 0;
        for(int i = 1; i <= a; i++) {
            for(int j = 1; j <= month[i-1]; j++) {
                day++;
                if(day%7 == 1) day%=7;
                if(i == a && j == b) break;
            }
        }
        return arr[day-1];
    }
}

public class year2016 {
    public static void main(String[] args) {
        Solution solution = new Solution();
        System.out.println(solution.solution(5, 24));
    }
}

File: Kotlin/Level1/year2016/year2016.kt
package Level1.year2016

class Solution {
    fun solution(a: Int, b: Int): String {
        var month = arrayOf(31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
        var date = arrayOf("FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU")
        var day = 0
        for(i in 0 until a-1) {
            day += month[i]
        }
        return date[(day + b - 1)%7]
    }
}

fun main(args: Array<String>) {
    val solution = Solution()
    println(solution.solution(5, 24))
}